examples/pxe_mapping_file.csv

```diff
@@ -1,3 +1,3 @@
 SERVICE_TAG,HOSTNAME,ADMIN_MAC,ADMIN_IP,BMC_IP
-6XCVT4,n1,xx:yy:zz:aa:bb:cc,10.5.0.101,10.3.0.101
-V345H5,n2,aa:bb:cc:dd:ee:ff,10.5.0.102,10.3.0.102
\ No newline at end of file
+ABCD12,n1,xx:yy:zz:aa:bb:cc,10.5.0.101,10.3.0.101
+ABCD34,n2,aa:bb:cc:dd:ee:ff,10.5.0.102,10.3.0.102
```

Comment:
Add return line at EOF

Reply:
Addressed changes in internal PR. It will be available in next set of PRs